TWLight/resources/models.py

```diff
@@ -423,6 +424,9 @@ def save(self, *args, **kwargs):
                     "Error trying to insert a tag: the JSON is invalid"
```

Comment:
For bonus points (not a blocker for merging) Since we're making this more usable, let's give the person a link to the list of available tags. Note that you'll also need to import mark_safe, eg
`from django.utils.safestring import mark_safe`
and update the tests.

```suggestion
                    mark_safe("Error trying to insert a tag: please choose a tag from <a rel='noopener' target='_blank' href='https://github.com/WikipediaLibrary/TWLight    /blob/production/locale/en/tag_names.json'>tag_names.json</a>.")
```

Reply:
Hello @jsnshrmn, thanks for the suggestion I will work on it. I was also thinking about this when testing code manually because I don't know what is valid tags that I can give for testing functionality.

Reply:
Hello @jsnshrmn, I have updated required changes. Thank you.

Reply:
:+1: